lib/src/widgets/chat/ds_carrousel.widget.dart

```diff
@@ -0,0 +1,145 @@
+    this.onSelected,
+    this.onOpenLink,
+    DSMessageBubbleStyle? style,
+  })  : style = style ?? DSMessageBubbleStyle(),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return _buildCollection();
+  }
+
+  Widget _buildCollection() {
+    final children = <Widget>[];
+    final String typeCollection;
+
+    (content['itemType'].contains('select'))
+        ? typeCollection = 'select'
+        : typeCollection = 'container';
+
+    final items = content['items'];
+
+    for (var item in items) {
+      if (typeCollection == 'select') {
+        Map<String, dynamic>? header = item["header"];
+        List options = item["options"];
+
+        var listOptions = <DSDocumentSelectOption>[];
+
+        for (final option in options) {
+          listOptions.add(DSDocumentSelectOption.fromJson(option));
+        }
+
+        children.add(
+          SizedBox(
+            width: DSUtils.bubbleMinSize,
+            child: DSImageMessageBubble(
+              align: align,
+              url: header!["value"]["uri"],
+              title: header["value"]["title"],
+              text: header["value"]["text"],
+              appBarText: header["value"]["title"],
+              selectOptions: listOptions,
+              showSelect: true,
+              hasSpacer: false,
+              onSelected: onSelected,
+              onOpenLink: onOpenLink,
+              style: style,
+            ),
+          ),
+        );
+      } else {
+        final contentContainer = item['value'];
+        final index = items.indexOf(item);
+        final radius = [DSBorderRadius.topLeft, DSBorderRadius.bottomLeft];
+
+        if (index == 0 &&
+            borderRadius.any((element) => [
+                  DSBorderRadius.topRight,
+                  DSBorderRadius.all
+                ].any((border) => element == border))) {
+          radius.add(DSBorderRadius.topRight);
+        } else if (index == items.length - 1 &&
+            borderRadius.any((element) => [
+                  DSBorderRadius.bottomRight,
+                  DSBorderRadius.all
+                ].any((border) => element == border))) {
+          radius.add(DSBorderRadius.bottomRight);
+        }
+
+        children.add(
+          Padding(
+            padding: const EdgeInsets.only(top: 8.0),
+            child: DSCard(
+              type: item['type'],
+              content: contentContainer,
+              align: align,
+              style: style,
+              borderRadius: radius,
+              customerName: contentContainer['text'],
+              onSelected: onSelected,
+              onOpenLink: onOpenLink,
+            ),
+          ),
+        );
+      }
+    }
+
+    return typeCollection == 'select'
+        ? Padding(
+            padding: const EdgeInsets.only(top: 8.0),
+            child: SingleChildScrollView(
+              padding: const EdgeInsets.symmetric(horizontal: 16.0),
+              scrollDirection: Axis.horizontal,
```

Comment:
Running on Android, I'm getting this error:
`Failed assertion: line 243 pos 14: 'notification.metrics.axis == widget.axis': is not true`

After a little research, I've found this [comment](https://github.com/flutter/flutter/issues/107772#issuecomment-1219906155) which suggest the following change:

```suggestion
              scrollDirection: Axis.horizontal,
              physics: const BouncingScrollPhysics(),
```

Reply:
Done